dist/css/ams-map.css

```diff
@@ -0,0 +1,886 @@
+/**
```

Comment:
dit ziet er uit als build output, `.gitignore` en verwijderen?

Reply:
ah, nvm. Dit is dilemma van een git resource includen in je package.json. Een tijd terug was er geen andere optie. Maar is er misschien nu een post-install optie van NPM voor git resources beschikbaar? Waarschijnlijk buiten scope

Reply:
buiten scope voor nu